.ci/install_kata_kernel.sh

```diff
@@ -65,9 +65,10 @@ get_packaged_kernel_version() {
 }
 
 build_and_install_kernel() {
+	# Always build and install the kernel version found locally
 	info "Install kernel from sources"
 	pushd "${tmp_dir}" >> /dev/null
-	"${kernel_repo_dir}/kernel/build-kernel.sh" "setup"
+	"${kernel_repo_dir}/kernel/build-kernel.sh" "setup" -v "${kernel_version}"
```

Comment:
Also I am not sure how well bash will handle this,  I prefer set this as:
` ./build-kernel.sh -v "${kernel_version} setup`